test/unit/vl-map-delete-action.test.html

```diff
@@ -59,18 +62,18 @@
         });
       });
 
-      test('controle van de delete actie configuratie met custom stijl', (done) => {
+      test('controle van de delete actie configuratie met custom stijl', async () => {
         const map = fixture('vl-map-delete-action-fixture-custom-stijl');
         const styleElement = map.querySelector('vl-map-layer-style');
         const layer = map.querySelector('vl-map-features-layer').layer;
 
-        map.ready.then(() => {
-          const action = map.map.actions[0];
-          assert.deepEqual(action.layer, layer);
-          assert.deepEqual(action.style(), styleElement.style());
-          assert.equal(action.style().getFill().getColor(), '#FFE615');
-          done();
+        await map.ready;
+        assert.deepEqual(map.map.actions[0].layer, layer);
+        await awaitUntil(() => {
+          const deleteAction = map.map.actions[0];
+          return deleteAction && deleteAction.style && (typeof deleteAction.style === 'function') && deleteAction.style().getFill().getColor() === '#FFE615';
         });
```

Comment:
Kan je hier geen gebruik maken van het `style-changed` event?

Reply:
Nee want dat event is van vl-map-layer en niet van vl-map-delete-action (of een andere action). Ik zou het ook daar kunnen invoeren maar dan zou het puur zijn om dit te kunnen testen, lijkt me niet zo opportuun?